src/dlgProfilePreferences.cpp

```diff
@@ -1779,3 +1804,37 @@ void dlgProfilePreferences::slot_script_selected(int index)
         preview->setText(mpHost->getActionUnit()->getAction(itemId)->getScript());
     }
 }
+
+/*!
+ * \brief dlgProfilePreferences::slot_changeShowSpacesAndTabs
```

Comment:
Sorry, didn't notice the git commit comment. Documenting our functions is definitely good but I believe it should be a team decision as to which documentation platform we use.